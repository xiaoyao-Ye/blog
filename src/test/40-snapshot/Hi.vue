<script setup>

</script>

<template>
  <div>
    <h1 id="hi">
      Hi
    </h1>
  </div>
</template>

<style lang="scss" scoped>

</style>
