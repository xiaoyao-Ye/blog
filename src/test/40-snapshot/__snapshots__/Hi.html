<div>
  <h1 id="hi"> Hi </h1>
</div>